packages/vuetify/src/components/VSheet/VSheet.ts

```diff
@@ -1,70 +1,115 @@
-// @ts-nocheck
-/* eslint-disable */
-
 // Styles
 import './VSheet.sass'
 
-// Mixins
-import BindsAttrs from '../../mixins/binds-attrs'
-import Colorable from '../../mixins/colorable'
-import Elevatable from '../../mixins/elevatable'
-import Measurable from '../../mixins/measurable'
-import Roundable from '../../mixins/roundable'
-import Themeable from '../../mixins/themeable'
-
-// Helpers
-import mixins from '../../util/mixins'
-
-// Types
-import { VNode } from 'vue'
-
-/* @vue/component */
-export default mixins(
-  BindsAttrs,
-  Colorable,
-  Elevatable,
-  Measurable,
-  Roundable,
-  Themeable
-).extend({
-  name: 'v-sheet',
-
-  props: {
-    outlined: Boolean,
-    shaped: Boolean,
-    tag: {
-      type: String,
-      default: 'div',
-    },
-  },
+// Utilities
+import { computed, defineComponent, h } from 'vue'
+import { convertToUnit } from '@/util/helpers'
+import makeProps from '@/util/makeProps'// Utilities
+import propsFactory from '@/util/propsFactory'
 
-  computed: {
-    classes (): object {
-      return {
-        'v-sheet': true,
-        'v-sheet--outlined': this.outlined,
-        'v-sheet--shaped': this.shaped,
-        ...this.themeClasses,
-        ...this.elevationClasses,
-        ...this.roundedClasses,
-      }
-    },
-    styles (): object {
-      return this.measurableStyles
-    },
-  },
+// Composables
+import {
+  makeBorderRadiusProps,
+  makeDimensionProps,
+  makeElevationProps,
+  makeTagProps,
+  useBorderRadius,
+  useDimension,
+  useElevation,
+  useTheme,
+} from '@/composables'
+
+const makePositionProps = propsFactory({
+  absolute: Boolean,
+  bottom: [Boolean, String],
+  fixed: Boolean,
+  left: [Boolean, String],
+  right: [Boolean, String],
+  top: [Boolean, String],
+})
+
+function usePosition (props: any) {
+  const positionStyles = computed(() => {
+    const styles: any = {}
+    const top = (props.top == null || props.top === true) ? '0' : props.top
+    const right = (props.right == null || props.right === true) ? '0' : props.right
+    const bottom = (props.bottom == null || props.bottom === true) ? '0' : props.bottom
+    const left = (props.left == null || props.left === true) ? '0' : props.left
+    const position = props.fixed ? 'fixed' : props.absolute ? 'absolute' : ''
+
+    console.log(top)
+
+    if (top) styles.top = convertToUnit(top)
+    if (right) styles.right = convertToUnit(right)
+    if (bottom) styles.bottom = convertToUnit(bottom)
+    if (left) styles.left = convertToUnit(left)
+    if (position) styles.position = position
+
+    return styles
+  })
+
+  return { positionStyles }
+}
+
+// Composables
+const makeOutlineProps = propsFactory({
+  outlined: [Boolean, String],
+})
+
+function useOutline (props: any) {
+  const outlined = props.outlined
```

Comment:
This isn't reactive.

`const outlined = toRef(props, 'outlined')` or access `props.outlined` in the computed.